lib/Hooks/UserHooks.php

```diff
@@ -200,6 +200,10 @@ public function postDeleteUser($params) {
 		if (App::isEnabled('encryption')) {
 			$this->keyManager->deletePublicKey($params['uid']);
 		}
+
+		if (!is_null($params['uid']) && ($params['uid'] !== '')) {
```

Comment:
I think it's better to move this inside the `App::isEnabled('encryption')` check, just for consistency. Maybe we can remove the `App::isEnabled('encryption')` check and remove the keys in any case. It's weird that some keys are removed if the encryption is disabled but others not.

If this is the right behaviour, add a comment explaining.

Reply:
Moved the piece of code inside `App::isEnabled('encryption')` check. Since its a hook, imho it would not be a good idea to remove the check `App::isEnabled('encryption')`.